google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineAction.java

```diff
@@ -138,22 +138,26 @@ protected File createApplicationDefaultCredentials() {
         GCLOUD_USER_TYPE_LABEL, GCLOUD_USER_TYPE
     );
     String jsonCredential = new Gson().toJson(credentialMap);
-    File tempCredentialFilePath = null;
+    File tempCredentialFilePath;
     try {
       tempCredentialFilePath = FileUtil
           .createTempFile(
               "tmp_google_application_default_credential",
               "json",
               true /* deleteOnExit */);
       Files.write(jsonCredential, tempCredentialFilePath, Charset.forName("UTF-8"));
+      credentialsPath = tempCredentialFilePath;
```

Comment:
Not a big deal, but I just wonder why you added this here as `credentialsPath` will be set anyway after returning from this function.